app/src/main/java/org/jellyfin/androidtv/model/itemtypes/LiftedItems.kt

```diff
@@ -4,20 +4,23 @@ import org.jellyfin.apiclient.model.dto.BaseItemDto
 import org.jellyfin.apiclient.model.dto.BaseItemType
 import org.jellyfin.apiclient.model.dto.ChapterInfoDto
 import java.util.*
+import kotlin.properties.Delegates
 
-sealed class BaseItem(original: BaseItemDto) {
+sealed class BaseItem(original: BaseItemDto) : ObservableParent() {
 	val id: String = original.id
 	val name: String = original.name
 	val description: String? = original.overview
 	val images: ImageCollection = ImageCollection(original)
 	val added: Date = original.dateCreated
+	var favorite: Boolean by Delegates.observable(original.userData.isFavorite, ::observer)
 }
 
 sealed class PlayableItem(original: BaseItemDto) : BaseItem(original) {
-	val canResume: Boolean = original.canResume
 	val playbackPositionTicks: Long = original.userData.playbackPositionTicks
+	val canResume: Boolean = original.canResume && playbackPositionTicks > 0
```

Comment:
I think the `playbackPositionTicks > 0` check should be sufficient, since the `original.canResume` just does a basically equivalent check.

```java
	public final boolean getCanResume()
	{
		return getUserData() != null && getUserData().getPlaybackPositionTicks() > 0;
	}
```


https://github.com/jellyfin/jellyfin-apiclient-java/blob/master/library/src/main/java/org/jellyfin/apiclient/model/dto/BaseItemDto.java#L2311

Reply:
I didn't even look at the baseItem. I assumed it was just passed from the server!